ads/zergnet.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, checkData, validateDataExists} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function zergnet(global, data) {
+    checkData(data, ['zergid']);
+    validateDataExists(data, ['zergid']);
+    global.zergnetWidgetId = data.zergid;
```

Comment:
you'll need to add these two properties here https://github.com/ampproject/amphtml/blob/master/ads/ads.extern.js